tools/test

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
I wondered whether this should just be a new command in `sigh` itself. For @csilvestrini as another `bazel` compatriot to comment.

Is the intent that this is a script a developer can run to essentially emulate locally in a more performant matter what we then run on CI?

Reply:
> Is the intent that this is a script a developer can run to essentially emulate locally in a more performant matter what we then run on CI?

Yes, this was my intention. After this would land, I'd try to advocate to change people's habits. To your first point, habit changing is hard, so it might be best to port this shell script into `tools/sigh test`.

Reply:
I like this idea. It'd be great if there was a way to run the standard battery of tests locally before uploading a PR. I'd have called it `presubmit` instead of `test`, but either is fine.

As to the question of whether it should be in `sigh` instead: I think philosophically sigh should be for handling our TypeScript/JavaScript stuff, and bazel for everything else. (It'd be really nice if bazel just did everything, but that's a separate issue for another day.) Currently, bazel uses sigh for various parts of its compilation/test process, e.g. it runs `sigh build`, and runs certain .ts unit tests using a special flag to `sigh test`. Things get confusing if we let sigh invoke bazel which invokes sigh.

So I guess what I'm saying is, it feels cleaner/simpler to me to keep this as a separate script, and leave sigh strictly for handling JS stuff.